tests/tests_pytorch/trainer/logging_/test_train_loop_logging.py

```diff
@@ -621,11 +622,26 @@ def training_step(self, *args):
 
     trainer = Trainer(default_root_dir=tmpdir, fast_dev_run=1)
     model = TestModel()
-    match = escape(f"self.log(foo, {value})` was called")
+    match = escape(f"self.log(foo, {value})` was called, but the tensor must have a single element")
```

Comment:
note: it is hard to find tests corresponding to source code when only a tiny fraction of the error sentence is exposed in the tests.

Reply:
We expose a tiny fraction because we want to test only that the warning/error is shown, not the language used inside. It's inconvenient when you make a small grammatical change and then the tests fail. Fun fact, [expecttests](https://github.com/ezyang/expecttest) address this issue